--- tsgo-baseline
+++ rust-actual
@@ -0,0 +1,17 @@
+assignmentCompatWithObjectMembersNumericNames.ts(20,1): error TS2741: Property '1' is missing in type 'T' but required in type 'S'.
+assignmentCompatWithObjectMembersNumericNames.ts(21,1): error TS2741: Property '1.' is missing in type 'S' but required in type 'T'.
+assignmentCompatWithObjectMembersNumericNames.ts(23,1): error TS2741: Property '1' is missing in type '{ 1.0: string; }' but required in type 'S'.
+assignmentCompatWithObjectMembersNumericNames.ts(25,1): error TS2741: Property '1' is missing in type 'T2' but required in type 'S2'.
+assignmentCompatWithObjectMembersNumericNames.ts(26,1): error TS2741: Property '1.0' is missing in type 'S2' but required in type 'T2'.
+assignmentCompatWithObjectMembersNumericNames.ts(27,1): error TS2741: Property '1' is missing in type 'T' but required in type 'S2'.
+assignmentCompatWithObjectMembersNumericNames.ts(28,1): error TS2741: Property '1' is missing in type '{ 1.0: string; baz?: string; }' but required in type 'S2'.
+assignmentCompatWithObjectMembersNumericNames.ts(29,1): error TS2741: Property '1' is missing in type '{ 1.0: string; }' but required in type 'S2'.
+assignmentCompatWithObjectMembersNumericNames.ts(31,1): error TS2741: Property '1.' is missing in type '{ 1.0: string; baz?: string; }' but required in type '{ 1.: string; bar?: string; }'.
+assignmentCompatWithObjectMembersNumericNames.ts(32,1): error TS2741: Property '1.0' is missing in type '{ 1.: string; bar?: string; }' but required in type '{ 1.0: string; baz?: string; }'.
+assignmentCompatWithObjectMembersNumericNames.ts(33,1): error TS2741: Property '1.' is missing in type 'S' but required in type '{ 1.: string; bar?: string; }'.
+assignmentCompatWithObjectMembersNumericNames.ts(34,1): error TS2741: Property '1.' is missing in type 'S2' but required in type '{ 1.: string; bar?: string; }'.
+assignmentCompatWithObjectMembersNumericNames.ts(35,1): error TS2741: Property '1.' is missing in type '{ 1.0: string; }' but required in type '{ 1.: string; bar?: string; }'.
+assignmentCompatWithObjectMembersNumericNames.ts(37,1): error TS2741: Property '1.0' is missing in type '{ 1: string; }' but required in type '{ 1.0: string; }'.
+assignmentCompatWithObjectMembersNumericNames.ts(38,1): error TS2741: Property '1' is missing in type '{ 1.0: string; }' but required in type '{ 1: string; }'.
+assignmentCompatWithObjectMembersNumericNames.ts(41,1): error TS2741: Property '1.0' is missing in type 'T' but required in type '{ 1.0: string; }'.
+